index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <div class="maincont">
+
+      <div class="app-bar">
+        <div class="upper-bar">
+          <p class="time">9 : 41</p> 
+          <div class="upper-icons">
+            <p class="ntew"> </p>
+            <p class="wifi"></p>
```

Comment:
- [ ] Kindly remove this section of your design as they were put on the Figma design as just a decoration 🙏 
![image](https://user-images.githubusercontent.com/74814780/185456862-962a128a-b666-4115-ac3f-3b62bad3299d.png)